apps/code-examples/src/app/v2/v2.component.ts

```diff
@@ -73,7 +73,7 @@ export default class CodeExamplesLandingComponent {
   constructor() {
     for (const [packageName, config] of Object.entries(DOCS.packages)) {
       for (const group of Object.keys(config.groups)) {
-        this.documentationGroups.push(`${packageName}${SEPARATOR}${group}`);
+        this.documentationGroups.push(`${group}${SEPARATOR}${packageName}`);
```

Comment:
Switching the order so that the groups are easier to find when scanning the list.